doc/about/user-guidelines.md

```diff
@@ -0,0 +1,67 @@
+# `mybinder.org` Usage Guidelines
+
+This page details some guidelines and policies that we follow at ``mybinder.org``.
+
+:::{admonition,warning} When in doubt, open an issue
+If you're not sure whether your use-case is reasonable for `mybinder.org`, don't
+hesitate to [open an issue](https://github.com/jupyterhub/mybinder.org-deploy/issues/new?labels=impact&template=request_resources.md)
+to ask if your planned usage is OK.
+:::
+
+## Temporary Banning
+
+Temporary banning means that `mybinder.org` will stop building / serving Binder
+sessions for a given repository. This usually happens because of some
+undesired behavior with the repository. Some examples of behavior that could
+result in temporary banning include:
+
+* A large, unexpected spike in traffic that persists over time.
+* Sessions that routinely use a large amount of CPU.
+* Sessions that attempt to perform a lot of outgoing networking traffic.
+
+The `mybinder.org` team uses the following workflow in discussing / deciding
+when to temporarily ban a repository:
+
+* An issue is created to document why a repo (or group of repos) is being banned
+* A PR that implements the ban is opened
+* A "bans" tag notes any PRs that created a ban
+* The issue stays open until the ban is lifted
+
+If you are temporarily banned, contact us on the
+[Gitter Channel](https://gitter.im/jupyterhub/binder) or
+[Open an Issue](https://github.com/jupyterhub/mybinder.org-deploy/issues) to discuss
+how to un-ban the repository.
+
+## Running events with `mybinder.org`
+
+If you'd like to run an event with `mybinder.org` and expect more than 50 users
+to join at once, please [request an increase in quota for your repository](https://github.com/jupyterhub/mybinder.org-deploy/issues/new?labels=impact&template=request_resources.md).
+
+The Binder Team occasionally increases the number of sessions that can run for a single repository if it is being used for purposes that align with the mission of the Binder Project. However, there is no guarantee that this will be done for your event.
+
+## Maximum Concurrent Users for a Repository
+
+We don't want a single repository to dominate all of the traffic to Binder, so
+we've set a maximum limit of concurrent user sessions that point to the same
+Binder link. **The maximum number of simultaneous users for a given repo is 100**.
+If you think you have a *really* good reason for why this number should be
+higher, please [Open an Issue](https://github.com/jupyterhub/mybinder.org-deploy/issues).
```

Comment:
Should we link to the same issue template as in the previous question? We'd want a lot of the same information as in the event case but the goal would be a permanent increase in quota.

Maybe we can get around having two templates or having to explain the difference between a temporary increase and a permanent one by rewording this question to be just about "what is the limit?" with an instruction to follow the link to the previous question if you want an increase. I think I like this idea because it has never happened that someone wanted a permanent increase (those all came through informal channels)